charts/console/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: console
 description: >-
   deploys the plural console and additional dependencies, for use in bring-your-own-kube setups
 appVersion: 0.10.40
-version: 0.3.68
+version: 0.3.69
```

Comment:
don't bump this please, needs to be bumped in the release

Reply:
done